src/Company/Company.ts

```diff
@@ -49,26 +51,38 @@ export class Company {
     if (p.relatedFaction) this.relatedFaction = p.relatedFaction;
   }
 
+  get favor() {
+    return this.#favor;
+  }
+
+  /**
+   * Read comments on Faction.setFavor for more information.
+   *
+   * @param value
+   */
+  setFavor(value: number) {
+    if (Number.isNaN(value)) {
+      this.#favor = 0;
+      return;
+    }
+    this.#favor = clampNumber(value, 0, MaxFavor);
+  }
+
+  gainFavor(value: number) {
```

Comment:
I was expecting `gainFavor` to do something more involved, but it's literally just calling `setFavor`. So why do we need it? Having a regular setter and doing `this.favor += value` would do the same thing.